dask/array/utils.py

```diff
@@ -225,7 +226,18 @@ def assert_eq_shape(a, b, check_nan=True):
             assert aa == bb
 
 
-def _get_dt_meta_computed(x, check_shape=True, check_graph=True):
+def _check_chunks(x):
+    x = x.persist()
```

Comment:
In testing we typically use the synchronous scheduler by default.  Happy either way though.

Reply:
Oh yes, I'd meant to do that and forgot. Done now.